### PR TITLE
jfrog

### PR DESCRIPTION
**Pull request by mcovarr**
Hop to it!

### CODE REVIEW

File: release/release_workflow.wdl [hunk 1]
```diff
@@ -122,7 +122,7 @@ task wait_for_artifactory {
     
     command <<<
         checkIfPresent() {
-            isPresent=$(curl -s --head https://artifactory.broadinstitute.org/artifactory/simple/libs-release-local/org/broadinstitute/${repo}/${version}/ | head -n 1 | grep -q "HTTP/1.[01] [23]..")
+            isPresent=$(curl -s --head https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/org/broadinstitute/${repo}/${version}/ | head -n 1 | grep -q "HTTP/1.[01] [23]..")
```
Comment (mcovarr) [inline release/release_workflow.wdl]: I haven't tested this change

**Comment by geoffjentry** [comment]
url

:+1:

**Comment by Horneth** [comment]
👍

File: release/release_workflow.wdl (hunk 1 above)
Comment (Horneth) [inline release/release_workflow.wdl]:
```
$ curl -s --head https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/org/broadinstitute/wdl4s_2.12/0.12/ | head -n 1
HTTP/1.1 200 OK
```

Looks good to me

Comment (mcovarr) [inline release/release_workflow.wdl]: 🐸

**Pull request merged**